### PR TITLE
Added Secrets Provider Helm chart version 1.1.6

### PR DESCRIPTION
**Pull request by diverdane**
### Desired Outcome

A new version (`1.1.6`) of the Secrets Provider Helm chart is published and can be
downloaded and installed. This will keep the version of this Helm chart in sync
with the current release of the `cyberark/secrets-provider-for-k8s repository.

### Implemented Changes

The Secrets Provider Helm chart version 1.1.6 is published and can be
downloaded and installed. The changes incorporated into this version compared
to version 1.1.5 of the Secrets Provider Helm chart:

- Secrets Provider container image specification now includes a
  `docker.io/` prefix so that manifests work on recent OpenShift versions
  (e.g. OpenShift 4.8 or newer). The image specification now looks like this:

  ```
  image: docker.io/cyberark/secrets-provider-for-k8s
  ```

### Connected Issue/Story

CyberArk internal issue link: [ONYX-11981](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-11981)

### Definition of Done

- [x] New Helm chart tar file is added to the `docs/` subdirectory.
- [x] `reindex.sh` script is run to perform reindex for this Helm repo.

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [ ] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

File: docs/index.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   conjur-config-cluster-prep:
   - apiVersion: v2
-    created: "2021-09-30T01:19:08.23103-07:00"
+    created: "2021-10-31T12:32:34.199223-04:00"
```
Comment (jtuttle) [inline docs/index.yaml:5]: Why did all of these other dates get updated? That doesn't seem right, but I'm guessing it just happens based on the fact that all of them are the same.

File: docs/index.yaml [hunk 2]
```diff
@@ -294,7 +294,17 @@ entries:
...
   - apiVersion: v1
-    created: "2021-09-30T01:19:08.251425-07:00"
+    created: "2021-10-31T12:32:34.219133-04:00"
+    description: A Helm chart for deploying CyberArk Secrets Provider for Kubernetes
+    digest: a6801d0494ca59f79fa81b85d22f95f6082f2d9be73f3f0e8533b82bfcabb02d
+    home: https://github.com/cyberark/secrets-provider-for-k8s
+    icon: https://www.cyberark.com/wp-content/uploads/2015/12/cybr-aim.jpg
+    name: secrets-provider
+    urls:
+    - https://cyberark.github.io/helm-charts/secrets-provider-1.1.6.tgz
```
Comment (jtuttle) [inline docs/index.yaml:304]: Should this URL work yet?

**Review by jtuttle — CHANGES_REQUESTED** [review]

File: docs/index.yaml (hunk 2 above)
Comment (diverdane) [reply docs/index.yaml:304]: Not yet. The changes in this PR have to merge first.

Some background: The way that this works is that when this repo was first created, a [GitHub Pages site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) was created for the repo. Now, anything merged into the `docs/` subdirectory, that also has an entry in the `docs/index.yaml` is then publicly available at `https://cyberark.github.io/helm-charts/*`.

Unfortunately, this means we have to do a final test the link after the merge.

**Review by jtuttle — APPROVED** [review]

File: docs/index.yaml (hunk 1 above)
Comment (diverdane) [reply docs/index.yaml:5]: This is something that the `helm repo index ...` command does. I don't know if there's a flag we can add to change this behavior. Here's the command we use to generate these changes (from `reindex.sh`):
```
helm repo index docs --url https://cyberark.github.io/helm-charts
```

**Pull request merged**